dev/archery/archery/docker/core.py

```diff
@@ -417,3 +417,19 @@ def _push(service):
 
     def images(self):
         return sorted(self.config.hierarchy.keys())
+
+    def info(self, key_name, filters=None, prefix=' '):
+        output = []
+        for key, value in key_name.items():
+            if hasattr(value, 'items'):
+                temp_filters = filters
+                if key == filters or filters is None:
+                    output.append(f'{prefix}- {key}')
+                    # Keep showing this specific key
+                    # as parent matched filter
+                    temp_filters = None
+                output.extend(self.info(value, temp_filters, prefix + " "))
```

Comment:
Should probably use two spaces for better and more visible alignment?

Reply:
good idea, with that I can remove the `-` and have a more visible alignment:
```
$ archery docker info debian-go-cgo 
Service debian-go-cgo docker-compose config:
  image: ${REPO}:${ARCH}-debian-${DEBIAN}-go-${GO}-cgo
  build
    context: .
    dockerfile: ci/docker/debian-go-cgo.dockerfile
    cache_from: ['${REPO}:${ARCH}-debian-${DEBIAN}-go-${GO}-cgo']
    args
      base: ${REPO}:${ARCH}-debian-${DEBIAN}-go-${GO}
  shm_size: 2G
  volumes: ['.:/arrow:delegated', '${DOCKER_VOLUME_PREFIX}debian-ccache:/ccache:delegated']
  environment
    ARROW_GO_TESTCGO: 1
  command: /bin/bash -c "
  /arrow/ci/scripts/go_build.sh /arrow &&
  /arrow/ci/scripts/go_test.sh /arrow"
```